benchbuild/project.py

```diff
@@ -17,21 +17,26 @@
 """
 import copy
 import logging
+import typing as tp
 import uuid
 from abc import abstractmethod
 from functools import partial
 from os import getenv
-from typing import List, Mapping, Optional, Tuple, Type
 
 import attr
 from plumbum import ProcessExecutionError, local
 from pygtrie import StringTrie
 
+import benchbuild as bb
 from benchbuild import signals
+from benchbuild.downloads.base import BaseSource
+from benchbuild.downloads.variants import VariantContext
+from benchbuild.downloads.versions import BaseVersionGroup
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No name 'downloads' in module 'benchbuild'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)